src/main/resources/executions/examples/london-cycle-traffic-air-quality-meanvalue.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
These recipes need description.

Reply:
if i provide description they wont run due to comments in recipe. If you want i can provide

Reply:
Yes, please do.

Reply:
@lorenaqendro done